about.html

```diff
@@ -0,0 +1,119 @@
+                    </ul>
+                </div>
+            </div>
+        </header>
+        <div class="hero_wrap">
+            <section class="hero about">
+                <div class="hero-inner slideshow__text-content has-animated" data-animate>
+                    <h3>"Technology is the only water drinking which makes you more thirsty"</h3>
+                    <h2>YOUR PARTNER FOR THE NEXT BIG LEAP</h2>
+                    <div class="hero_msg">
+                        Two days of incredible masterclasses, workshops, speakers, tech leaders, leading tech brands,
+                        networking, opportunities, recruiters and more. This is set to be an unmissable event for anyone
+                        in Tech.
+                    </div>
+                    <div class="hero_location about">
+                        <div class="hero_date about">2023.01.11(Wed) ~ 12(Thursday)</div>
+                        <div class="hero_venue about">@San Francisco, CA & Virtual</div>
+                    </div>
+                </div>
+            </section>
+        </div>
+        <section class="aboutEvent">
+            <div class="event_wrapper">
+                <h2>About the Event</h2>
+                <div class="guide_bar"></div>
+                <p class="event_description">Digital transformation has touched every sphere of life. We are all
+                    immersed in a world of data. This data facilitates our healthcare, business and government
+                    activities, commerce and entertainment. The Techno TechoThirst Summit: Looking Forward highlights
+                    the ways in which society has been changed by digital disruption and provides the roadmap for what
+                    lies ahead.
+
+                    The biggest stories of the day – cryptocurrency, the Metaverse, supply chain management and
+                    cybersecurity – are all covered at this comprehensive and timely event. We will also learn about new
+                    developments in biotechnology, space travel, gaming, and work from home solutions.
+
+                    Through all of these conversations, we explore the new possibilities engendered by technology and
+                    the shape, look and feel of the future. We also touch upon the possible downside of unfettered
+                    growth of technology such as the increased vulnerability to cyberattacks, the degradation of
+                    privacy, and the inherent presence of biases that negatively impact marginalized groups in society.
+                </p>
+            </div>
+        </section>
+        <section class="secSectionAbout">
+            <article class="past_events">
+                <h2>See the past Nova Events</h2>
+                <div class="guide_bar"></div>
+                <p>Take a look at the past Nova events which took place in Dubai and in Erbil</p>
```

Comment:
- [ ] kindly add a logo in the mid section of the about page like in the original design. this will also add your skills in adding images within text.
yours
![Screenshot (734)](https://user-images.githubusercontent.com/65298048/211082162-62519c22-5b92-4872-9a22-9df4ee0c8080.png)
behance
![Screenshot (735)](https://user-images.githubusercontent.com/65298048/211082238-8283e324-76e3-43b6-ba9d-655881ab5815.png)

Reply:
added. Thanks.

<img width="597" alt="image" src="https://user-images.githubusercontent.com/39811497/211108906-bebe6072-444b-4eb2-a1c7-4c15a4be020f.png">